pages/lazer/how-lazer-works.mdx

```diff
@@ -1,5 +1,77 @@
+import { Callout } from "nextra/components";
+
 # How Pyth Lazer works
 
-Pyth Lazer is a permissioned service that provides ultra-low-latency price and market data to highly latency-sensitive users.
+Pyth Lazer is a permissioned service that provides ultra-low-latency market data to consumers.
+It aggregates data from multiple publishers and distributes it to consumers through a multi-tier architecture.
+
+## System Services
+
+The architecture consists of five main services that work together to provide ultra-low-latency data to consumers.
```

Comment:
Let's say "types of services" instead of just "services". Each of them has multiple nodes for redundancy (we can mention that here too).